conf.pri.mingw

```diff
@@ -0,0 +1,43 @@
+# Adapt these paths
+# Point these to the include folders
+INCLUDEPATH += $$quote(C:/qBittorrent/boost_1_64_0)
+INCLUDEPATH += $$quote(C:/qBittorrent/RC_1_1/include)
+INCLUDEPATH += $$quote(C:/qBittorrent/Zlib/include)
```

Comment:
the capitalized Z bothers me...
These are just some changes I would do, you can decide yourself.

Reply:
Fixed.